apps/console/src/features/claims/constants/claim-management-constants.ts

```diff
@@ -110,25 +112,25 @@ export class ClaimManagementConstants {
     ]
 
     public static readonly OIDC_MAPPING: string[] = [
-        "http://wso2.org/oidc/claim"
+        SCIMConfigs.oidc
     ];
 
     public static readonly SCIM_MAPPING: string[] = [
-        "urn:ietf:params:scim:schemas:core:2.0:User",
-        "urn:scim:schemas:core:1.0",
-        "urn:ietf:params:scim:schemas:core:2.0",
-        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"
+        SCIMConfigs.scim.userSchema,
+        SCIMConfigs.scim.core1Schema,
+        SCIMConfigs.scim.coreSchema,
+        SCIMConfigs.scim.enterpriseSchema
     ]
 
     public static readonly OIDC: string = "oidc";
     public static readonly SCIM: string = "scim";
     public static readonly OTHERS: string = "others";
 
     public static readonly SCIM_TABS = [
-        { name: "Enterprise Schema", uri: "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User" },
-        { name: "User Schema", uri: "urn:ietf:params:scim:schemas:core:2.0:User" },
-        { name: "Core Schema", uri: "urn:ietf:params:scim:schemas:core:2.0" },
-        { name: "Core 1.0 Schema", uri: "urn:scim:schemas:core:1.0" }
+        {name: "Enterprise Schema", uri: SCIMConfigs.scim.enterpriseSchema},
```

Comment:
Possible ESLint warnings here. Please check if ESLint plugin is working properly.

Reply:
Fixed in commit 709db97832e822082dff6db0d23962b5e6773072